xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1042,12 +1041,6 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
 
   // show art for shoutcast v2 streams (set to false for devices with limited storage)
   XMLUtils::GetBoolean(pRootElement, "shoutcastart", m_bShoutcastArt);
-
-  // movie fanarts
-  TiXmlElement* pFanart = pRootElement->FirstChildElement("fanart");
```

Comment:
Can you remove [fanart](https://kodi.wiki/view/Advancedsettings.xml#fanart) from the wiki when this PR is merged?

Reply:
Yes, had already tagged it to remind me. There are other wiki changes related to art to do as wel